app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,38 @@
+class ReviewsController < ApplicationController
+
+  def index
+    @reviews = Review.all
+  end
+
+  def new
+    @review = Review.new
+  end
+
+  def create
+    @review = Review.new({
+      user_id: review_params[:user][:id],
```

Comment:
Don't trust the front-end to send user id. Get it from current_user - anything from the front end can be faked.